pulsar-client/pom.xml

```diff
@@ -139,10 +144,6 @@
                   <shadedPattern>com.yahoo.pulsar.shade.org.apache.commons</shadedPattern>
                 </relocation>
                 <relocation>
-                  <pattern>org.asynchttpclient</pattern>
```

Comment:
If we don't rename the package, then we shouldn't include the classes in the shaded jar.. we should just leave it as a maven dependency.